pkg/network/driver_interface.go

```diff
@@ -130,6 +136,23 @@ func (di *DriverInterface) setupFlowHandle() error {
 	return nil
 }
 
+// SetFlowFilters installs the provided filters for flows
+func (di *DriverInterface) SetFlowFilters(filters []driver.FilterDefinition) error {
```

Comment:
Same for this? Should be on the `Handle` interface?

Reply:
Thank you for the suggestion. As discussed in Slack, will move to different PR as needed.